pipeline/export/nwb.py

```diff
@@ -42,92 +49,230 @@ def datajoint_to_nwb(session_key):
+                location=json.dumps([{k: v for k, v in stim_locs.items()
+                                    if k not in experiment.Photostim.primary_key}
+                                    for stim_locs in (experiment.Photostim.PhotostimLocation
+                                                    & photostim_key).fetch(as_dict=True)], default=str),
+                description=f'excitation_duration: {photostim["duration"]}')
+            nwbfile.add_ogen_site(stim_site)
+            stim_sites[photostim['photo_stim']] = stim_site 
+
+    # =============================== TRACKING =============================== 
+    # add tracking device 
+
+    tables = [(tracking.Tracking.NoseTracking,'nose'),(tracking.Tracking.TongueTracking,'tongue'),
+        (tracking.Tracking.JawTracking,'jaw'),(tracking.Tracking.LeftPawTracking,'left_paw'),
+        (tracking.Tracking.RightPawTracking,'right_paw'),(tracking.Tracking.LickPortTracking,'lickport'),
+        (tracking.Tracking.WhiskerTracking,'whisker')]
+
+    if tracking.Tracking & session_key:
+        behav_acq = pynwb.behavior.BehavioralTimeSeries(name='BehavioralTimeSeries')
+        nwbfile.add_acquisition(behav_acq)
+
+        for table, name in tables:
+            if table & session_key:
+                x, y, likelihood, samples, start_time, sampling_rate= (experiment.SessionTrial 
+                                                                    * tracking.Tracking 
+                                                                    * table 
+                                                                    * tracking.TrackingDevice.proj('sampling_rate') 
+                                                                    & session_key).fetch(f'{name}_x',f'{name}_y', 
+                                                                                            f'{name}_likelihood', 
+                                                                                            'tracking_samples','start_time',
+                                                                                            'sampling_rate',order_by='trial')
+
+                recording_times_length = samples/sampling_rate.astype(float)
+                unshifted_time_stamps = [np.linspace(0,x, samples[idx].astype(int)) for idx, x in enumerate(recording_times_length)]
+                filled_start_times = [np.full(len(unshifted_time_stamps[idx]),x) for idx, x in enumerate(start_time)]
+                shifted_time_stamps = [np.add(filled_start_times[x].astype(float),unshifted_time_stamps[x]) for x in range(len(filled_start_times))]
+
+                behav_acq.create_timeseries(name=f'BehavioralTimeSeries_{name}_x_y_data', 
+                                            data=np.vstack([np.hstack(x),np.hstack(y), np.hstack(likelihood)]),
+                                            timestamps=np.hstack(shifted_time_stamps),
+                                            description='video description',
```

Comment:
update `'video description'` with something else?

Reply:
changed video description to `description=f'Time series for {name} position: x, y, likelihood'`